futures-util/src/io/mod.rs

```diff
@@ -221,6 +223,16 @@ pub trait AsyncReadExt: AsyncRead {
     {
         split::split(self)
     }
+
+    /// Wraps an [`AsyncRead`] in a compatibility wrapper that allows it to be
+    /// used as a futures 0.1 / tokio-io 0.1 `AsyncRead`.
```

Comment:
Maybe mention that this will simultaneously support `AsyncWrite` if implemented for bidirectional IO?

Reply:
Done.